packages/esm-patient-banner-app/src/banner-tags/active-visit-tag.component.tsx

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import styles from './active-visit-tag.scss';
 import { useTranslation } from 'react-i18next';
-import { Tag, TooltipDefinition } from 'carbon-components-react';
+import { InlineLoading, Tag, TooltipDefinition } from 'carbon-components-react';
 import { formatDatetime, parseDate, useVisit } from '@openmrs/esm-framework';
 interface ActiveVisitBannerTagProps {
   patientUuid: string;
   patient: fhir.Patient;
 }
 const ActiveVisitBannerTag: React.FC<ActiveVisitBannerTagProps> = ({ patientUuid, patient }) => {
   const { t } = useTranslation();
-  const { currentVisit } = useVisit(patientUuid);
+  // @ts-ignore
```

Comment:
To be remove once [https://github.com/openmrs/openmrs-esm-core/pull/332](https://github.com/openmrs/openmrs-esm-core/pull/332) is merged in